python/hsml/model.py

```diff
@@ -297,14 +297,16 @@ def experiment_project_name(self, experiment_project_name):
         self._experiment_project_name = experiment_project_name
 
     @property
-    def path(self):
-        """path of the model."""
-        path = "Models/{}/{}".format(self.name, str(self.version))
-        if self._shared_registry_project_name is not None:
-            path = path.replace(
-                "Models", "{}::Models".format(self._shared_registry_project_name)
-            )
-        return path
+    def model_path(self):
+        """path of the model with version folder omitted. Resolves to /Projects/{project_name}/Models/{name}"""
+        return "/Projects/{}/Models/{}".format(self.project_name, self.name)
+
+    @property
+    def version_path(self):
+        """path of the model including version folder. Resolves to /Projects/{project_name}/Models/{name}/{version}"""
+        return "/Projects/{}/Models/{}/{}".format(
+            self.project_name, self.name, str(self.version)
+        )
```

Comment:
Could we reuse _model_path_?

```suggestion
        return "{}/{}".format(self.model_path, str(self.version))
```

Reply:
That's a good idea I'll do that